packages/plans-grid-next/src/components/shared/plan-actions.tsx

```diff
@@ -22,16 +22,16 @@ import styled from '@emotion/styled';
 import { useSelect } from '@wordpress/data';
 import { useCallback } from '@wordpress/element';
 import { TranslateResult, useTranslate } from 'i18n-calypso';
-import { usePlansGridContext } from '../grid-context';
-import useDefaultStorageOption from '../hooks/data-store/use-default-storage-option';
-import useIsLargeCurrency from '../hooks/use-is-large-currency';
-import { useManageTooltipToggle } from '../hooks/use-manage-tooltip-toggle';
-import { usePlanPricingInfoFromGridPlans } from '../hooks/use-plan-pricing-info-from-grid-plans';
+import { usePlansGridContext } from '../../grid-context';
+import useDefaultStorageOption from '../../hooks/data-store/use-default-storage-option';
+import useIsLargeCurrency from '../../hooks/use-is-large-currency';
+import { useManageTooltipToggle } from '../../hooks/use-manage-tooltip-toggle';
+import { usePlanPricingInfoFromGridPlans } from '../../hooks/use-plan-pricing-info-from-grid-plans';
 import PlanButton from './plan-button';
-import { Plans2023Tooltip } from './plans-2023-tooltip';
-import type { GridPlan, PlanActionOverrides } from '../types';
+import { PlansTooltip } from './plans-tooltip';
```

Comment:
I think similar too. `Plans` just feels ambiguous to me. Perhaps also overly saturated/used polymorphically.